express-notes-api/data.json

```diff
@@ -0,0 +1,29 @@
+{
+  "nextId": 12,
+  "notes": {
+    "2": {
+      "id": 2,
+      "content": "The event loop, this, closures, and prototypal inheritance are special about JavaScript."
+    },
+    "5": {
+      "id": 5,
+      "content": "In JavaScript, the value of `this` is determined when a function is called; not when it is defined."
+    },
+    "6": {
+      "id": 6,
+      "content": "A closure is formed when a function retains access to variables in its lexical scope."
+    },
+    "9": {
+      "id": 9,
+      "content": "Inertia is a property of matter."
+    },
+    "11": {
+      "id": 11,
+      "content": "I am still awake at this time."
+    },
+    "undefined": {
```

Comment:
Hmm, looks like a bug that inserted this while you were developing! There's no bug now, though!

And this is good evidence that you actually worked on the code and didn't just copy/paste! That's awesome!